client/community/action-creators/dns-control/async-add-dns-record.js

```diff
@@ -0,0 +1,23 @@
+import * as t from '../../action-types'
+import { createAction } from '../create-action'
+import * as CommunitySelectors from '../../selectors'
+
+import AuthSelectors from '~client/account/redux/selectors'
+
+export default ({ dns_hosted_zone_id, ...dnsRecord }) => (dispatch, getState, { api }) => {
```

Comment:
Variables like `dns_hosted_zone_id` defined using **snake_case** will produce **standardjs** lint warnings. To check if the lint are passing, you can run the following command on root of the project:

```
./node_modules/.bin/standard ./client/**/*.js // <~ change the folder is necessary
```

And to fix some warning automatically:
```
./node_modules/.bin/standard --fix ./client/**/*.js // <~ change the folder is necessary
```